src/modules/google-organization-kp/includes/Rest_Controller.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+namespace Wordlift\Modules\Google_Organization_Kp;
+
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use Wordlift_Configuration_Service;
+
+class Rest_Controller {
+
+    public function register_hooks() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	public function register_routes() {
+        register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/pages',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'list_pages' ),
+				'permission_callback' => '__return_true',
+			)
+		);
+
+		register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/countries',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'list_countries' ),
+				'permission_callback' => '__return_true',
+			)
+		);
+
+		register_rest_route(
+			WL_REST_ROUTE_DEFAULT_NAMESPACE,
+			'/wl-google-organization-kp/field-data',
+			array(
+				'methods'             => 'GET',
+				'callback'            => array( $this, 'existing_data' ),
+				'permission_callback' => '__return_true',
```

Comment:
We should make it more restrictive, maybe `manage_options`.

Reply:
Done.